internal/config/config.go

```diff
@@ -15,11 +15,16 @@ type Service struct {
 	Address string `mapstructure:"address"`
 }
 
+type Server struct {
+	ShutdownTimeout int `mapstructure:"shutdown_timeout"` // in minutes
```

Comment:
use native time.Duration type and time.Duration-formatted values in the config

Reply:
done